src/app/public/modules/page/page-theme.service.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Injectable
+} from '@angular/core';
+
+@Injectable()
+export class SkyPageThemeService {
+
+  private styleEl: HTMLStyleElement;
+
+  public addTheme() {
+    if (!this.styleEl) {
+      this.styleEl = document.createElement('style');
```

Comment:
Should the logic for manipulating DOM elements be moved to a DOM adapter? (See `SkyTextExpandAdapterService` for reference.)

Reply:
That was the spirit of this service. I just didn't put the word "Adapter" in the name. I'll fix it.